docs/Targets.md

```diff
@@ -0,0 +1,123 @@
+# Targets
+
+## SiFive HiFive1 RISC-V
+
+### Features
+* E31 RISC-V 320MHz 32-bit processor
+* Onboard 16KB scratchpad RAM
+* External 4MB QSPI Flash 
+
+### Default Linker Settings
+* FLASH: Address 0x20000000, Len 0x6a120 (424 KB)
+* RAM:   Address 0x80000000, Len 0x4000  (16 KB)
+
+### Stock bootloader
+Start Address: 0x20000000 is 64KB. Provides a "double tap" reset feature to halt boot and allow debugger to attach for reprogramming. Press reset button, when green light comes on press reset button again, then board will flash red.
+
+### Application Code
+Start Address: 0x20010000
+
+### wolfBoot configuration
+
+The default wolfBoot configuration will add a second stage bootloader, leaving the stock "double tap" bootloader for safety.
```

Comment:
'for safety' ... it's not clear to me how leaving the doubletap as stage1 increases safety

Reply:
also, from the addresses seems like you are actually replacing it